public/css/widget/quick-actions.less

```diff
@@ -2,31 +2,44 @@
   display: flex;
   flex-wrap: wrap;
   list-style-type: none;
-  // TODO: Eh? Minus .25em on the left and right, maybe, because padding and responsiveness.
-  //       But what is the additional 1em on the left for???
-  margin: 0 -1.25em 0 -.25em;
-  margin: 0;
+  margin: 0 -.5em;
+  width: auto;
   padding: 0;
 
   a {
     text-decoration: none;
   }
 
-  li {
-    .rounded-corners();
-    margin-right: 1em;
+  a,
+  button {
     padding: .25em;
-    vertical-align: middle;
-    white-space: nowrap;
+    .rounded-corners();
+    display: inline-block;
 
     &:hover {
       background: @gray-lighter;
     }
   }
+
+  li {
+    margin: 0 .25em .5em .25em;
```

Comment:
This leads to a strange alignment for me: (notice that the icons in default layout are indented differently than in wide layout)

### default layout
![Screenshot from 2021-07-13 12-40-25](https://user-images.githubusercontent.com/16668527/125438643-307ed3b2-d23f-414e-8021-fb0d008135a5.png)

### wide layout
![Screenshot from 2021-07-13 12-40-41](https://user-images.githubusercontent.com/16668527/125438657-2a93af1d-db5f-4a32-a2e7-1774a7ded5ab.png)

Reply:
This is interesting. Because in Safari this looks completely as intended (see screenshots). I'll have a look at the other browsers.

Reply:
Should be fixed with  8f763c7

✅